github/resource_github_repository.go

```diff
@@ -44,6 +44,12 @@ func resourceGithubRepository() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"visibility": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				Default:      "public",
+				ValidateFunc: validateValueFunc([]string{"public", "private", "internal"}),
```

Comment:
I think you could also use `validation.StringInSlice()` here though this works :)

Reply:
Ah nice. I haven't browsed the validation methods at all - grabbed this one from another resource. This looks much more meaningful.